src/reducers/projects.js

```diff
@@ -209,6 +209,19 @@ export default function reduceProjects(stateIn, action) {
         action.meta.timestamp,
       );
 
+    case 'PROJECT_EXPORTED':
```

Comment:
We probably want to consume this in the `projects` saga and update the data in Firebase in this case too, right?